package.json

```diff
@@ -2,23 +2,21 @@
   "name": "nodeapi",
   "version": "1.3.0",
   "description": "node api for COINS",
-  "main": "index.js",
+  "main": "dist/index.js",
   "directories": {
     "test": "test"
   },
-  "engines": {
-    "iojs": "2.5.0"
-  },
   "dependencies": {
     "axios": "0.5.4",
+    "babel": "5.8.23",
     "bcrypt": "0.8.5",
     "bluebird": "2.10.0",
     "bookshelf": "0.8.2",
-    "bookshelf-shield": "2.0.0",
+    "bookshelf-shield": "git+https://github.com/MRN-Code/bookshelf-shield",
```

Comment:
Thanks for doing that. Let's get it in place before merging this. I've published v2.1.1, and added you as a collaborator.